src/features/identity/IdentityAgentManager.ts

```diff
@@ -70,7 +73,27 @@ const isFirstLaunch = async () => {
 };
 
 const startAgent = async (passphrase: string) => {
-  return await agent.start({ passphrase });
+  await agent.start({ passphrase });
+  await startSync();
+};
+
+const startSync = async () => {
+  // Register all DIDs under management, as well as the agent's master DID
+  const managedIdentities = await agent.identityManager.list();
+  const didsToRegister = [
+    agent.agentDid,
+    ...managedIdentities.map((i) => i.did),
+  ];
+
+  await Promise.all(
+    didsToRegister.map((did) => agent.syncManager.registerIdentity({ did }))
+  );
+
+  // TODO: Once selective sync is enabled, only sync for records that the mobile identity agent
+  // cares about. We DO NOT want to sync every record the user has in their DWN to their mobile device.
+  agent.syncManager.startSync({ interval: ms("2m") }).catch((error: any) => {
+    console.error(`Sync failed: ${error}`);
+  });
```

Comment:
Created a task to implement this once it's available: https://github.com/TBD54566975/web5-wallet/issues/104